artic-ncov2019/2.0.0/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL container.version="1"
+LABEL software="ARTIC-nCov19"
+LABEL software.version="2.0.0"
+LABEL description="A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore"
+LABEL website="https://github.com/artic-network/fieldbioinformatics"
+LABEL license="https://github.com/artic-network/fieldbioinformatics/blob/master/LICENSE"
+LABEL maintainer="Abigail Shockey"
+LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+
+# install needed software for conda install
+RUN apt-get update && apt-get install -y \
+  wget \
+  git \
+  build-essential
```

Comment:
Could you please append this to this line? Will help keep the docker image a bit smaller
```
build-essential && \
apt-get autoclean && rm -rf /var/lib/apt/lists/*
```

Reply:
Also added!